text/0059-otlp-trace-data-format.md

```diff
@@ -0,0 +1,346 @@
+  SpanKind kind = 7;
+
+  // start_time_unixnano is the start time of the span. On the client side, this is the time
+  // kept by the local machine where the span execution starts. On the server side, this
+  // is the time when the server's application handler starts running.
+  //
+  // This field is semantically required and it is expected that end_time >= start_time.
+  //
+  // This field is required.
+  int64 start_time_unixnano = 8;
+
+  // end_time_unixnano is the end time of the span. On the client side, this is the time
+  // kept by the local machine where the span execution ends. On the server side, this
+  // is the time when the server application handler stops running.
+  //
+  // This field is semantically required and it is expected that end_time >= start_time.
+  //
+  // This field is required.
+  int64 end_time_unixnano = 9;
+
+  // attributes is a collection of attribute key/value pairs. The value can be a string,
+  // an integer, a double or the Boolean values `true` or `false`. Note, global attributes
+  // like server name can be set as tags using resource API. Examples of attributes:
+  //
+  //     "/http/user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36"
+  //     "/http/server_latency": 300
+  //     "abc.com/myattribute": true
+  //     "abc.com/score": 10.239
+  repeated AttributeKeyValue attributes = 10;
+
+  // dropped_attributes_count is the number of attributes that were discarded. Attributes
+  // can be discarded because their keys are too long or because there are too many
+  // attributes. If this value is 0, then no attributes were dropped.
+  int32 dropped_attributes_count = 11;
+
+  // TimedEvent is a time-stamped annotation of the span, consisting of either
+  // user-supplied key-value pairs, or details of a message sent/received between Spans.
+  message TimedEvent {
+    // time_unixnano is the time the event occurred.
+    int64 time_unixnano = 1;
+
+    // name is a user-supplied description of the event.
+    string name = 2;
+
+    // attributes is a collection of attribute key/value pairs on the event.
+    repeated AttributeKeyValue attributes = 3;
+
+    // dropped_attributes_count is the number of dropped attributes. If the value is 0,
+    // then no attributes were dropped.
+    int32 dropped_attributes_count = 4;
+  }
+
+  // timed_events is a collection of TimedEvent items.
+  repeated TimedEvent timed_events = 12;
+
+  // dropped_timed_events_count is the number of dropped timed events. If the value is 0,
+  // then no events were dropped.
+  int32 dropped_timed_events_count = 13;
+
+  // Link is a pointer from the current span to another span in the same trace or in a
+  // different trace. For example, this can be used in batching operations,
+  // where a single batch handler processes multiple requests from different
+  // traces or when the handler receives a request from a different project.
```

Comment:
You can also point to the overview document describing links

Reply:
Done.